### PR TITLE
Home page complete API integration, fix event model

### PR DESCRIPTION
**Pull request by evgiz**
Home page content is now fully provided by api.
Also fixes lots of issues with event data model.

### CODE REVIEW

File: frontend/src/api.ts [hunk 1]
```diff
@@ -54,6 +55,13 @@ export async function getUser(): Promise<UserDto> {
   return response.data;
 }
 
+export async function getHomeData(): Promise<HomePageElementDto[]> {
+  const url = BACKEND_DOMAIN + ROUTES.backend.samfundet__home;
+  const response = await axios.get<HomePageElementDto[]>(url, { withCredentials: true });
+
+  return response.data;
+}
```
Comment (emilte) [inline frontend/src/api.ts:63]: returnere response?

File: frontend/src/dto.ts [hunk 2]
```diff
@@ -67,7 +77,7 @@ export type EventDto = {
   title_en: string;
   start_dt: Date;
   duration: number;
-  end_dt?: Date;
+  end_dt: Date;
```
Comment (emilte) [inline frontend/src/dto.ts:80]: sjekk

**Review by emilte — APPROVED** [review]

**Review by Mathias-a — APPROVED** [review]
LOKTM

File: frontend/src/dto.ts (hunk 2 above)
Comment (evgiz) [reply frontend/src/dto.ts:80]: Se #433

**Pull request merged**